.template-helpers/variant.sh

```diff
@@ -23,9 +24,16 @@ for tag in "${tags[@]}"; do
 	fi
 done
 if [ "$text" ]; then
-	# give a little space
-	echo
-	echo
-	cat "$dir/variant.md"
-	echo "$text"
+  latest=($(curl -sSL $url | sed -e 's/git:\/\/github.com\///' -e 's/@/ /' - | grep "latest"))
+  dockerfile='https://raw.githubusercontent.com/'"${latest[1]}"'/'"${latest[2]}"'/'"${latest[3]}"'/Dockerfile'
```

Comment:
I changed the grep to `grep "latest.*github.com"` and then do a `if [ -z "$latest" ]; then exit 0; fi`